Lesson07/HW07.Pyramid/HW07.Pyramid/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace HW07.Pyramid
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Дан массив 2d 4 на 5
+            int i=4, j=5;
+            int[,] array = new int[i, j];
+
+            //Измените значения массива так, чтобы при выводе в консоль массив имел пирамиду из единиц
+            array[1, 2] = 1; array[2, 1] = 1; array[2, 2] = 1; array[2, 3] = 1;array[3, 0]=1; 
+            array[3, 1] = 1; array[3, 2] = 1; array[3, 3] = 1; array[3, 4] = 1;
```

Comment:
What are you doing here? 1 What is the number?  You need to use cycles only. you chose not correct logic.